app-misc/flirc/flirc-3.9.12.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit unpacker
+
+DESCRIPTION="Allows you to pair any remote control with your computer or media center"
+HOMEPAGE="https://flirc.tv/"
+SRC_URI="https://packagecloud.io/Flirc/repo/packages/ubuntu/zesty/flirc_${PV}_amd64.deb/download.deb -> ${P}.deb"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+IUSE="+qt5"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}"
+
+DEPEND=""
+RDEPEND="virtual/libusb
+	dev-libs/hidapi
+	qt5? (
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtgui:5
+		dev-qt/qtxml:5
+		dev-qt/qtxmlpatterns:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtcore:5 )"
+
+src_prepare () {
+	eapply_user
+	if use qt5 ; then
+		sed -i -e 's/^Version=.*$/Version=1.0/' "${S}"/usr/share/applications/Flirc.desktop || die
+	fi
+}
+
+src_install () {
+	insinto /lib/udev/rules.d
+	newins etc/udev/rules.d/99-flirc.rules 51-flirc.rules
+	doman usr/share/doc/flirc/flirc_util.1
+	exeinto /opt/bin
```

Comment:
There is no need to use `/opt` here. It's a common misunderstanding but `/opt` is meant for packages that do not adhere to `/usr` structure and e.g. need data files below executable, or are otherwise strongly separate.